config/config.md

```diff
@@ -132,7 +132,7 @@ The following config option are provided by the OpenHIM. All of these options ha
     "orchestrationTSBufferMillis": 100
   },
   "statsd": {
-    // Rather use a statsd server for reporting metrics
+    // Report application metrics and transaction metrics to a statsd server
```

Comment:
Aah ok think I answered my own question after reading through. So we're getting rid of statsd support on the metrics API, but keeping the existing reporting to statsd if someone wants to use it...